src/main/java/org/terasology/staticCities/raster/standard/RectPartRasterizer.java

```diff
@@ -46,7 +33,7 @@ protected void raster(RasterTarget brush, RectBuildingPart part, HeightMap heigh
         }
 
 //        int topHeight = part.getBaseHeight() + part.getWallHeight() + part.getRoof().getHeight;
-//        Region3i bbox = Region3i(rc.minX(), part.getBaseHeight(), rc.minY(), rc.maxX(), topHeight, rc.maxY());
+//        BlockRegion bbox = new BlockRegion(rc.minX(), part.getBaseHeight(), rc.minY(), rc.maxX(), topHeight, rc.maxY());
 
 //        if (chunk.getRegion().overlaps(bbox)) {
```

Comment:
this this commented out code relevant?

Reply:
honestly? no idea

Reply:
let's clean this up in the ... well ... module cleanup.